pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "linz_logger"
-version = "0.9.0"
+version = "0.1.0"
```

Comment:
this version is wrong. it should be `0.10.0`

Reply:
Fixed.